src/components/SearchForm/SearchForm.js

```diff
@@ -1,51 +1,60 @@
-import React, { useState } from 'react'
-import "./SearchForm.css"
-import EventsContainer from "../EventsContainer/EventsContainer"
-
-const SearchForm = ({setSearchedEvents}) => {
-const [artistName, setArtistName] = useState("");
-const [results, setResults] = useState([])
-
-const handleChange = (e) => {
-  setArtistName(e.target.value)
-
-}
-
-
-const handleClick = (e) => {
-  e.preventDefault()
-
-  fetch(`https://concert-crew-be.herokuapp.com/api/v1/events?keyword=${artistName}`)
-  .then(response => response.json())
-  .then(data => {
-    setResults(data.data.events)
-    setSearchedEvents(data.data.events)
-  })
-  setArtistName('')
-}
-
-
-const toBeDisplayed = results.length ?  
-<EventsContainer events={results}/> 
-: <p>Search for an upcoming show!</p>
+import React, { useState } from "react";
+import "./SearchForm.css";
+import EventsContainer from "../EventsContainer/EventsContainer";
+
+const SearchForm = ({ setSearchedEvents }) => {
+  const [artistName, setArtistName] = useState("");
+  const [results, setResults] = useState([]);
+  const [message, setMessage] = useState("Search for an upcoming show!");
+
+  const handleChange = (e) => {
+    setArtistName(e.target.value);
+  };
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    artistName
+      ? fetch(
```

Comment:
I'd like to see fetch calls abstracted to a utility file, or at least another method in this component - can be hard to reach when the fetch is conditionally firing based on a ternary value

Reply:
Done!